pom.xml

```diff
@@ -62,7 +62,7 @@
         <dep.nexus-staging-plugin.version>1.6.8</dep.nexus-staging-plugin.version>
         <dep.asm.version>6.2.1</dep.asm.version>
         <dep.gcs.version>1.9.17</dep.gcs.version>
-        <dep.alluxio.version>2.1.2</dep.alluxio.version>
+        <dep.alluxio.version>2.3.0-SNAPSHOT</dep.alluxio.version>
```

Comment:
Do they have a stable release now? It will be easy to review if they have a release.

Reply:
In a day or two, alluxio will cut the release.